infra/docker/keycloak/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+sed -i "s;dashboard_url;$DASHBOARD_CLIENT_URL;g" /opt/keycloak/themes/phrasea/email/html/password-reset.ftl
+
+
+sed -i "s;dashboard_url;$DASHBOARD_CLIENT_URL;g" /opt/keycloak/themes/phrasea/email/text/password-reset.ftl
+
+/opt/keycloak/bin/kc.sh start --optimized
```

Comment:
Entrypoint should not incluse the command, try to run the command from here like you tried with bash